Configurations/unix-Makefile.tmpl

```diff
@@ -196,22 +196,23 @@ CXXFLAGS={- join(' ', @{$config{cxxflags}}) -}
 LDFLAGS= {- join(' ', @{$config{lflags}}) -}
 PLIB_LDFLAGS= {- join(' ', @{$config{plib_lflags}}) -}
 EX_LIBS= {- join(' ', @{$config{ex_libs}}) -}
-LIB_CPPFLAGS={- join(' ',
-                     (map { '-D'.$_ }
-                          ('OPENSSLDIR="\"$(OPENSSLDIR)\""',
-                           'ENGINESDIR="\"$(ENGINESDIR)\""')),
-                     $target{shared_cppflag}) || "" -}
-LIB_CFLAGS={- $target{shared_cflag} || "" -}
-LIB_CXXFLAGS={- $target{shared_cxxflag} || "" -}
-LIB_LDFLAGS={- $target{shared_ldflag}." ".$config{shared_ldflag} -}
-DSO_CPPFLAGS={- $target{dso_cppflags} || "" -}
-DSO_CFLAGS={- $target{dso_cflags} || "" -}
-DSO_CXXFLAGS={- $target{dso_cxxflags} || "" -}
-DSO_LDFLAGS={- $target{dso_lflags} || "" -}
-BIN_CPPFLAGS={- $target{bin_cppflags} || "" -}
-BIN_CFLAGS={- $target{bin_cflags} || "" -}
-BIN_CXXFLAGS={- $target{bin_cxxflags} || "" -}
-BIN_LDFLAGS={- $target{bin_lflags} || "" -}
+
+LIB_CPPFLAGS=$(CPPFLAGS){- join(' ',
+                                $target{shared_cppflag},
```

Comment:
I don't quite follow how does this work with respect to space after $(CPPFLAGS). I mean if I just run ./config the resulting line is `LIB_CPPFLAGS=$(CPPFLAGS) -DOPENSSLDIR=...`, but where does space come from. It looks like it works because $target{shared_cppflag} is undefined in mine attempt. But that would mean that if it was defined, then the outcome would be wrong, i.e. `LIB_CPPFLAGS=$(CPPFLAGS)-something -DOPENSSLDIR=...`.

Reply:
Ah, good point.  Will fix

Reply:
As a matter of fact, Appveyor shows us exactly that kind of issue...